workflow/scripts/run_readii.py

```
@@ -159,17 +264,82 @@ def main(dataset_index:pd.DataFrame,
+                                for neg_image, transform, region in image_types
                             )
 
+    return sample_results, dataset_feature_dir
+
+
+
+def main(dataset_index:pd.DataFrame,
+         pyrad_params:str,
+         procdata_path:Path,
+         results_path:Path,
+         regions:list[str] = ["roi", "non_roi", "full"],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)